cicecore/cicedynB/general/ice_flux.F90

```diff
@@ -33,19 +33,21 @@ module ice_flux
 
       !-----------------------------------------------------------------
       ! Dynamics component
+      ! All variables are assumed to be on the atm or ocn thermodynamic
+      ! grid except as noted
       !-----------------------------------------------------------------
 
       real (kind=dbl_kind), dimension (:,:,:), allocatable, public :: &
 
        ! in from atmos (if .not.calc_strair)  
-         strax   , & ! wind stress components (N/m^2)
-         stray   , &
+         strax   , & ! wind stress components (N/m^2), on grid_atm_dynu
+         stray   , & !                                 on grid_atm_dynv
 
        ! in from ocean
-         uocn    , & ! ocean current, x-direction (m/s) on T grid from forcing, should it be on U??
-         vocn    , & ! ocean current, y-direction (m/s) on T grid from forcing
-         ss_tltx , & ! sea surface slope, x-direction (m/m)
-         ss_tlty , & ! sea surface slope, y-direction
+         uocn    , & ! ocean current, x-direction (m/s) on T grid from forcing, on grid_ocn_dynu
+         vocn    , & ! ocean current, y-direction (m/s) on T grid from forcing, on grid_ocn_dynv
```

Comment:
Shouldn't the 'on T grid from forcing' be removed in the comments here?

Reply:
Good catch, I added the two comments at different times and wasn't careful to review when I added the second.  I fixed this.